test/verify/check-system-services

```diff
@@ -218,13 +236,14 @@ Unit=test.service
         b.wait_not_in_text("#services-list", "test")
         self.wait_service_state("basic.target", "active")
 
-        # Make sure that symlinked services also appear in the list
-        self.wait_service_state("reboot.target", "inactive")
-        self.wait_service_state("ctrl-alt-del.target", "inactive")
-        self.wait_service_state("runlevel6.target", "inactive")
-        self.goto_service("runlevel6.target")
-        b.wait_in_text(".service-name", "Reboot")
-        b.click("#services-page .pf-c-breadcrumb__link")
+        if not user:
```

Comment:
Let's also test some user units -- `dbus.service` should be a pretty safe bet. Also, the user unit has systemd's standard targets, like `basic.target` and `default.target`. The latter two also exist on the system instance, so they can be tested for both values of `user`.

Reply:
These checks where about to check symlinked units. These three uints are essentially the same file. I was not able to find symlinked units for system.